app/workers/education_form/templates/1995.erb

```diff
@@ -29,13 +29,6 @@ Preferred Method of Contact: <%= @applicant.preferredContactMethod %>
                   TYPE AND PROGRAM OF EDUCATION OR TRAINING
                   -----------------------------------------
 
-Benefit You Are Receiving: <%= form_benefit %>
```

Comment:
Do you want this line gone?  It's not STEM specific and [was present before the STEM stuff was added](https://github.com/department-of-veterans-affairs/vets-api/blame/71b87a85784cc236710717dc823d8b6ab2266a28/app/workers/education_form/templates/1995.erb).

Reply:
I think that we do want to add that back in, I will do so.